e2e/test/scenarios/custom-column/cc-help-text.cy.spec.js

```diff
@@ -34,24 +33,26 @@ describe("scenarios > question > custom column > help text", () => {
   });
 
   it("should not appear when formula field is not in focus (metabase#15891)", () => {
-    enterCustomColumnDetails({ formula: "rou{enter}1.5" });
+    enterCustomColumnDetails({ formula: "rou{enter}1.5){leftArrow}" });
```

Comment:
It seems we've changed the behavior here since the time #15891 was originally posted.

Previously it was enough to type `rou`, then press enter and see the `round(|)` with the cursor in the middle.
Now we don't close the parentheses anymore.